### ISSUE TITLE
Add Segger Real Time Terminal (RTT) support to J-Link plug-in

### ISSUE THREAD

**Issue by ilg-ul** [gnuarmeclipse/plug-ins#31]
As a faster alternative to semihosting, [RTT](https://www.segger.com/jlink-real-time-terminal.html) provides a bidirectional channel between the target and the host, that can be used to send trace messages.

**Comment by metc** [comment]
Yes, it would be great if [the RTT terminal](https://www.segger.com/jlink-rtt.html) could be included in Eclipse. I currently use the CF Terminal (Console) View plugin, as described here: https://mcuoneclipse.com/2015/04/20/serial-terminal-view-in-eclipse-luna/

**Comment by ilg-ul** [comment]
I am still considering it, but still had no time to investigate how complicated the implementation might be.

**Comment by gonzzor** [comment]
Any more ideas on this? The RTT Console is basically just a Telnet connection to JLinkGDBServer. Another option would be to launch JLinkRTTClient.

**Comment by ilg-ul** [comment]
as long as there are workarounds, the priority will probably remain low.

**Comment by gonzzor** [comment]
Thanks.
